markdown_preview/main_container.py

```diff
@@ -278,6 +279,9 @@ def _on_reload_unsafe(self):
 			return
 		start, end = doc.get_bounds()
 		unsaved_text = doc.get_text(start, end, True)
+		cursor_position = doc.get_property("cursor-position")
+		text_iter = doc.get_iter_at_offset(cursor_position)
+		self._webview_manager.set_cursor_position (text_iter.get_line()+1, text_iter.get_line_offset()+1)
```

Comment:
why "+1"? does it not understand zeros? then isn't it problematic that you set cursor_row and column as 0 in the init?